packages/preview/psl-thesis/0.1.0/typst.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "psl-thesis"
```

Comment:
If this package is officially endorsed by the university staff, it would need to undergo a short verification process. If it is not, could you please change the name to something more unique? You can simply add a distinctive adjective to the current name if you lack inspiration.

Reply:
The official LaTeX template from this university was originally created by a student. I'll check with them quickly if they can endorse this one, and if not I'll change the name!

Reply:
A member of the university staff will have to send an email to `hello@typst.app` stating that this package is official.

Reply:
Perfect, I'll shoot them an email and keep you updated!